app/src/main/java/org/mozilla/fenix/wallpapers/WallpaperManager.kt

```diff
@@ -134,6 +142,28 @@ class WallpaperManager(
         }
     }
 
+    fun recordDiscoveredMetric() {
+        val hasSentMetric = settings.wallpapersDiscovered
+        if (!hasSentMetric) {
+            metrics.track(Event.Wallpaper.DiscoveredFeature)
+            settings.wallpapersDiscovered = true
+        }
+    }
+
+    private fun recordWallpaperAppliedMetric(appliedWallpaper: Wallpaper) {
+        metrics.track(Event.Wallpaper.NewWallpaperApplied(appliedWallpaper))
```

Comment:
Run-by: Maybe we should inline this since it's only called once, consists of a single line in the function, and it's generally well understood what this `metrics.track()` does.

Reply:
Inlined